gradle/formatting.gradle

```diff
@@ -81,6 +82,12 @@ allprojects {
             paddedCell()
           }
         }
+        format 'misc', {
+            target '*.md', '*.gradle', '**/*.yaml', '**/*.yml', '**/*.svg'   
```

Comment:
FYI, might want to audit the file extensions in this repo.  I added `.svg` over on SDK because we have them in some of our design docs, but not sure it's needed here.

Reply:
We do have svg [here](https://github.com/opensearch-project/OpenSearch/blob/main/.idea/icon.svg) as well. Anyway, I feel it's good to have.